### PR TITLE
feat(dapps): revamp connected dApps screen

### PR DESCRIPTION
**Pull request by nistadev**
**Screenshots**

_No Dapps_
image

_With Dapps_
image

_Dapp hover_
image

### CODE REVIEW

**Comment by neuodev** [comment]
[autoscirpt] Jira Ticket: **[Connected DApps Page](https://emurgo.atlassian.net/browse/YOEXT-681)**

[](__JIRA_COMMENT_FLAG)

File: packages/yoroi-extension/app/components/dapp-connector/ConnectedWebsites/ConnectedWebsitesPage.js [hunk 1]
```diff
@@ -1,20 +1,24 @@
...
-import { connectorMessages } from '../../../i18n/global-messages';
-import { isErgo } from '../../../api/ada/lib/storage/database/prepackaged/networks';
-import WalletRow from './WalletRow';
 import type { TokenRow } from '../../../api/ada/lib/storage/database/primitives/tables';
 import type { TokenLookupKey, MultiToken } from '../../../api/common/lib/MultiToken';
 import type { ConceptualWalletSettingsCache } from '../../../stores/toplevel/WalletSettingsStore';
 import type { WalletChecksum } from '@emurgo/cip4-js';
+import { Component } from 'react';
+import { observer } from 'mobx-react';
+import { ReactComponent as NoItemsFoundImg } from '../../../assets/images/dapp-connector/no-websites-connected.inline.svg';
```
Comment (neuodev) [inline packages/yoroi-extension/app/components/dapp-connector/ConnectedWebsites/ConnectedWebsitesPage.js]:
```suggestion
import { ReactComponent as NoDappsFoundImg } from '../../../assets/images/dapp-connector/no-dapps-connected.inline.svg';
```

File: packages/yoroi-extension/app/components/dapp-connector/ConnectedWebsites/ConnectedWebsitesPage.js [hunk 2]
```diff
@@ -37,6 +45,10 @@ const messages = defineMessages({
     id: 'connector.connect.noWebsitesConnected',
     defaultMessage: "!!!You don't have any websites connected yet",
   },
+  connectedDapps: {
+    id: 'connector.connected-dapps.title',
+    defaultMessage: '!!!Connected DApps ({numDapps})',
```
Comment (neuodev) [inline packages/yoroi-extension/app/components/dapp-connector/ConnectedWebsites/ConnectedWebsitesPage.js]:
```suggestion
    defaultMessage: '!!!Connected DApps ({dappsCount})',
```

File: packages/yoroi-extension/app/i18n/locales/en-US.json [hunk 3]
```diff
@@ -64,6 +64,7 @@
   "connector.connect.info": "Your connection preferences will be saved to your Yoroi dApp list.",
   "connector.connect.noWalletsFound": "Ooops, no {network} wallets found.",
   "connector.connect.noWebsitesConnected": "You don't have any websites connected yet",
+  "connector.connected-dapps.title": "Connected DApps ({numDapps})",
```
Comment (neuodev) [inline packages/yoroi-extension/app/i18n/locales/en-US.json]:
```suggestion
  "connector.connected-dapps.title": "Connected DApps ({dappsCount})",
```

**Review by neuodev — CHANGES_REQUESTED** [review]
Wallet information is missing. 

Screenshot 2023-09-21 at 11 16 39 AM

**Comment by neuodev** [comment]
> Wallet information is missing.
> 
> Screenshot 2023-09-21 at 11 16 39 AM

Fixed 👇 

<details><summary>Preview</summary>
<p>

Screenshot 2023-09-21 at 11 31 15 AM


</p>
</details>

**Comment by neuodev** [comment]
Note: flow errors in this PR are caused by the "target" branch. should be fixed their first.

**Review by neuodev — APPROVED** [review]

**Review by zuzunker — APPROVED** [review]

**Review by zuzunker — COMMENTED** [review]
/check

**Pull request merged**